packages/grid/x-data-grid/src/models/api/gridDensityApi.ts

```diff
@@ -15,14 +15,6 @@ export interface GridDensityApi {
   /**
    * Sets the density of the grid.
    * @param {string} density Can be: `"compact"`, `"standard"`, `"comfortable"`.
-   * @param {number} headerHeight The new header height.
-   * @param {number} rowHeight The new row height.
-   * @param {number} maxDepth The depth of maximal depth column header grouping tree.
    */
-  setDensity: (
-    density: GridDensity,
-    headerHeight?: number,
-    rowHeight?: number,
-    maxDepth?: number,
```

Comment:
I'm not sure what is `maxDepth` used. I wasn't able to find any usage in the code.

Reply:
Yeah, seems to be unused :+1:

Reply:
Yes, that's a legacy from the initial implementation of column grouping